src/locale/locales/de/messages.po

```diff
@@ -1166,48 +1108,48 @@ msgstr "Erledigt{extraText}"
 
 #: src/view/com/auth/login/ChooseAccountForm.tsx:45
 msgid "Double tap to sign in"
-msgstr ""
+msgstr "Doppeltippen zum Anmelden"
 
 #: src/view/screens/Settings/index.tsx:755
 msgid "Download Bluesky account data (repository)"
-msgstr ""
+msgstr "Laden Sie Ihre Bluesky-Kontodaten herunter (Repository)"
```

Comment:
I remembered that there were two other places that `repository` is used. Here, the context is that this string is an `accessibilityHint` describing what clicking/tapping the `Export My Data` link in the Settings screen does. I think in this instance it's OK to reword and expand a bit to explain that it opens a modal to export your data, it doesn't actually start downloading anything straight away.

This is the DeepL translation of: `Opens a modal to download your Bluesky account data (repository)`. Does that make sense?

```suggestion
msgstr "Öffnet ein Modal zum Herunterladen deiner Bluesky-Kontodaten (Kontodepot)"
```

Reply:
I like it!

Reply:
Nice 👌